runtime/doc/plugins.txt

```diff
@@ -157,4 +158,42 @@ tohtml({winid}, {opt})                                       *tohtml.tohtml()*
         (`string[]`)
 
 
+==============================================================================
+Builtin plugin: difftool                                            *difftool*
+
+
+:DiffTool {left} {right}                                           *:DiffTool*
+Compares two files or directories side-by-side in Neovim.
+Supports directory diffing, rename detection, and highlights changes
+in quickfix list.
+
+The plugin is not loaded by default; use `:packadd nvim.difftool` before
+invoking `:DiffTool`.
+
+Example `git difftool -d` integration using `DiffTool` command: >ini
+    [difftool "nvim_difftool"]
+      cmd = nvim -c "packadd nvim.difftool" -c "DiffTool $LOCAL $REMOTE"
+    [diff]
+      tool = nvim_difftool
+<
+
+
+close()                                                     *difftool.close()*
+    Close the diff layout and clean up
```

Comment:
why is this needed? why can't it handle BufDelete and/or other events? this `close()` function is already implicitly assuming "the current buffer" so it's not even that useful anyway.

Reply:
hmm, i just had it there because i kinda liked just doing :DiffTool to also close but yea its not super useful and its better if it cleans up when one of the wins is closed